client/src/components/Button/DoubleActionButton.jsx

```diff
@@ -0,0 +1,12 @@
+import './DoubleActionButton.scss';
+import { Link } from 'react-router-dom';
+
+export const DoubleActionButton = ({leftText, leftLinkTo, rightText, rightLinkTo }) => {
+    return (
+        <div className="double-button">
+            <Link to={leftLinkTo}>{leftText}</Link>
```

Comment:
we need to account for the `onClick` property of the buttons throughout the app, since they won't just link to another page; the parent component can provide the function that should be called when the button is clicked, so that should be reflected in the props the button will take

I think we also need to account for the `type` property of the button; again this can be determined by whichever parent component calls the button, but it should be reflected in the props